File: examples/python/simple.py
import sys
from pathlib import Path
from monocr_onnx import read_image

# Path to test image
project_root = Path(__file__).resolve().parent.parent.parent
image_path = project_root / "data/images/test_0005_h71.png"

print(f"Reading image: {image_path}")
text = read_image(image_path)

print("Recognized Text:")
print(text)

File: examples/js/simple.js
import { read_image } from 'monocr';
import path from 'path';
import { fileURLToPath } from 'url';

const __filename = fileURLToPath(import.meta.url);
const __dirname = path.dirname(__filename);

async function main() {
    const modelPath = path.join(__dirname, '../../model/monocr.onnx');
    const charsetPath = path.join(__dirname, '../../model/charset.txt');
    const imagePath = path.join(__dirname, '../../data/images/test_0005_h71.png');

    console.log('MonOCR Simple Example\n');
    console.log('Reading image:', path.basename(imagePath));
    
    const text = await read_image(imagePath);
    console.log('\nRecognized text:');
    console.log(text);
}

main().catch(console.error);
